distributed/comm/ucx.py

```diff
@@ -596,14 +603,25 @@ def _scrub_ucx_config():
         if dask.config.get("distributed.comm.ucx.nvlink"):
             tls = tls + ",cuda_ipc"
 
-        options = {"TLS": tls, "SOCKADDR_TLS_PRIORITY": tls_priority}
+        high_level_options = {"TLS": tls, "SOCKADDR_TLS_PRIORITY": tls_priority}
+
+    # Pick up any other ucx environment settings
+    # {"some-name": value} is translated to {"SOME_NAME": value}
+    def normalise(k):
+        return "_".join(map(str.upper, k.split("-")))
+
+    environment_options = {
+        normalise(k): v
+        for k, v in dask.config.get("distributed.comm.ucx.environment", {}).items()
+    }
 
-    # ANY UCX options defined in config will overwrite high level dask.ucx flags
```

Comment:
Many years ago (#3515) we could sort of do what I'm introducing, but that was removed as part of a cleanup in #4850 (when this comment became out of date but was not updated)